geant3.sh

```diff
@@ -34,10 +35,11 @@ module-whatis "ALICE Modulefile for $PKGNAME $PKGVERSION-@@PKGREVISION@$PKGHASH@
 # Dependencies
 module load BASE/1.0 ROOT/$ROOT_VERSION-$ROOT_REVISION
 # Our environment
-setenv GEANT3_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
-setenv GEANT3DIR \$::env(GEANT3_ROOT)
-setenv G3SYS \$::env(GEANT3_ROOT)
-prepend-path PATH \$::env(GEANT3_ROOT)/bin
-prepend-path LD_LIBRARY_PATH \$::env(GEANT3_ROOT)/lib64
-$([[ ${ARCHITECTURE:0:3} == osx ]] && echo "prepend-path DYLD_LIBRARY_PATH \$::env(GEANT3_ROOT)/lib64")
+set GEANT3_ROOT \$::env(BASEDIR)/$PKGNAME/\$version
+setenv GEANT3_ROOT \$GEANT3_ROOT
+setenv GEANT3DIR \$GEANT3_ROOT
```

Comment:
Do we really need those environment variables?Why not simply G3SYS?

Reply:
GEANT3_ROOT is used in geant3_vmc-examples.sh, fairroot.sh, o2.sh. I think, G3SYS and GEANT3DIR can be dropped (but it will need to be tested)